res/skins/Shade/effectunit.xml

```diff
@@ -15,40 +15,54 @@
 				</Style>
 				<BackPath>style/style_bg_effectunit</BackPath>
 				<Children>
-					<WidgetGroup>
-						<Pos>9,5</Pos>
-						<Size>170,90</Size>
-						<Layout>vertical</Layout>
+					<HighlightingGroup>
```

Comment:
Also, `HighlightingGroup` isn't a great name considering any QSS property could be changed. How about `SelectionGroup` and `SelectionGroupContainer`?